gltfTutorial/gltfTutorial_004_ScenesNodesCamerasAnimations.md

```diff
@@ -0,0 +1,273 @@
+# Scenes, nodes, cameras and animations

```

Comment:
After reading this section, I suggest considering restructuring the tutorial.  It seems like a reader would have to go through the entire tutorial in order to get started with glTF.  This section, for examples, goes into animations (_before_ accessors) and cameras, both of which are not needed to get started.

Perhaps it would be useful to restructure so that the initial goal is to build up to a minimal glTF asset (e.g., no cameras, materials, animations, etc), and then start adding complexity.  As is, I don't know that the reader has enough context unless (1) they read the tutorial in its entirety or (2) they already have experience with glTF or another non-trivial 3D format.